front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/Settings.tsx

```diff
@@ -0,0 +1,43 @@
+import { type ChangeEvent } from 'react';
+
+import { Checkbox } from '@osrd-project/ui-core';
+import { X } from '@osrd-project/ui-icons';
+import { useTranslation } from 'react-i18next';
+
+type Settings = {
+  showConflicts: boolean;
+};
+
+type SettingsProps = {
+  settings: Settings;
+  onChange: (settings: Settings) => void;
+  onClose: () => void;
+};
+
+const Settings = ({ settings, onChange, onClose }: SettingsProps) => {
+  const { t } = useTranslation('simulation');
+
+  const handleConflictsChange = (event: ChangeEvent<HTMLInputElement>) => {
+    onChange({ ...settings, showConflicts: event.target.checked });
+  };
+
+  return (
+    <div id="settings-panel" className="flex justify-end absolute">
+      <div className="settings-panel-section">
+        <div className="settings-panel-section-title">{t('timeSpaceChartSettings.paths')}</div>
```

Comment:
I'm not sure, but did you reuse classes from ui-speedspacechart ?

```suggestion
      <div>
        <div>{t('timeSpaceChartSettings.paths')}</div>
```

Reply:
Yes, the classes are re-used. I left a note about this in Matrix: this is far from ideal, especially since some of these CSS rules are using the HTML `id` attribute. The alternative would be to duplicate the CSS (and keep both in sync when the mockup changes). Could do that if you prefer.

On the long run, I think it'd make sense to refactor the settings panel CSS inside osrd-ui to allow for code re-use.

Reply:
The speedspacechart classes are no longer re/ab-used.